### PR TITLE
Added: Okta SSO integration config guide

### PR DESCRIPTION
**Pull request by mattezra**

### CODE REVIEW

**Review by bltarkany — APPROVED** [review]

**Comment by mattezra** [comment]
Put on hold:

Config guide will be updated once Okta SSO implementation is done

**Comment by mattezra** [comment]
@shrunyan this is the Okta SSO config guide that will be published in zesty.org, content is the same with the [Okta SSO Configuration Guide](https://docs.google.com/document/d/1Qw865uR8nN_8gHiY2TTMFaSnAVbbjv_HefGyIVH9UYM/edit?pli=1#).

File: guides/integrations/okta-sso-and-zesty.io.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    
+1. Login to https://www.zesty.io/ and open developer tools in browser
+2. Go to the ***"Application"*** tab and navigate to ***"Cookies"*** and open https://www.zesty.io
+3. See **APP_SID** cookie and copy the cookie value (see image below)
+
+<figure><img src="../../.gitbook/assets/okta-config-guide-get-token.png" alt="Get token"><figcaption></figcaption></figure>
+        
+### **Get ecosystem_zuid**
+
+1. Use Zesty’s [Get Ecosystems API](https://accounts-api.zesty.org/#750eb50b-c53c-44aa-aa4a-b0b1199fb4bf) as seen below, replacing the token for Authorization (obtained in get token step)
```
Comment (shrunyan) [inline guides/integrations/okta-sso-and-zesty.io.md:41]: @mattezra we need a note here letting them know this requires have access to an instance. i.e. you can't get an ecosystem zuid unless you have access to an instance in an ecosystem

File: guides/integrations/okta-sso-and-zesty.io.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+4. Get your organization’s **Okta domain** (or **Issuer URL**) which can be found in the top-right dropdown button under your email
+    * Domain SHOULD CONTAIN **https://** i.e. https://org.okta.com
+5. Assign people or group who can use the Zesty.io SSO application in ***"Assign"*** section
+6. Zesty admin MUST register their organization’s **Client ID**, **Client Secret** and **Client Domain** in Zesty
+    * The Zesty user who will register the organization’s secrets MUST be an existing Zesty user with **admin** or **owner** role
+    * 3 steps to accomplish this: get token, get ecosystem_zuid and register
+    
+### **Get token**
+    
+1. Login to https://www.zesty.io/ and open developer tools in browser
```
Comment (shrunyan) [inline guides/integrations/okta-sso-and-zesty.io.md:33]: I wish we had a better way to provide a users session token to them

**Review by shrunyan — COMMENTED** [review]
couple of improvements

**Comment by mattezra** [comment]
@shrunyan done updating the documentation with your feedback.

@giseleblair 

[Here's](https://app.gitbook.com/o/-LKUXrO7WutbL3Y2yCzM/s/-LKUXrO8I7u5ufn3Lefn/~/change-requests/followUp) the link to the change request in Gitbook.

**Pull request merged**